backend/app/lifespan.py

```
@@ -0,0 +1,61 @@
+from contextlib import asynccontextmanager
+import asyncio
+import logging
+from sqlalchemy import text
+from fastapi import FastAPI
+from common.db.connection import engine
+
+logger = logging.getLogger(__name__)
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.14.8)

Unused function argument: `app`

(ARG001)